test/Microsoft.SqlTools.Migration.IntegrationTests/Migration/MigrationServiceTests.cs

```diff
@@ -42,8 +54,8 @@ public async Task TestHandleMigrationAssessmentRequest()
             }
         }
 
-        [Test]
-        [Ignore("Disable failing test")]
+        [NUnit.Framework.Test]
+        [NUnit.Framework.Ignore("Disable failing test")]
```

Comment:
I think Importing in this way will sove the issue, check it out
using NUnit.Framework;
using Assert_ = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;

Reply:
done